File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        
        total = 0
        for i in operations:
            if i == '++X' or i == 'X++':
                total = total + 1
            if i == '--X' or i == 'X--':
                total = total - 1 
        return total 

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int total = 0;
        for (int i = 0; i < operations.length; i++){
            if (operations[i].equals("++X") || operations[i].equals("X++")){
                total += 1;
            }else{
                total -= 1;
            }
        }
        return total;
    }
}